samples/documentation/lulesh_sample1_unix_commented.yaml

```diff
@@ -187,10 +191,41 @@ study:
       # on 'run-lulesh_*' which declares dependency on ALL combinations of
       # 'run-lulesh').
 
+####################################
+# Parameter Block
+####################################
+# The parameter block contains all the things we'd like to vary in the study.
+# Currently, there are two modes of operating in the specification:
+# 1. If a parameter block is specified, the study is expanded and considered a
+#   parameterized study.
+# 2. If a parameter block is not specified, the study is treated as linear and
+#    the resulting study is not expanded.
+#
+# There are three keys per parameter:
+# 1. A list of values that the parameter takes.
+# 2. A label that represents a "pretty printed" version of the parameter. The
+#    parameter values is specfied by the '%%' moniker (for example, for SIZE --
+#    when SIZE is equal to 10, the label will be 'SIZE.10').
+# 3. (Optional) A list of custom names for each value in the values list.

```

Comment:
If `names` is specified is that used instead of the parameter `values` when creating the workspace directories?

Reply:
This comment prompted me to go back and realize I was not remembering the functionality correctly. This'll be fixed in the next update.